src/Views/listings/create.phtml

```diff
@@ -23,25 +36,37 @@ $title = 'Create Listing';
     </div>
     <div class="my-4 mb-3">
         <label for="description">Description</label>
-        <textarea class="form-control" name="description" id="description" rows="4" placeholder="Type here..."></textarea>
+        <textarea type="text" class="form-control" name="description" id="description" rows="4" minlength="5" maxlength="300" placeholder="Type here..."><?= $params['description'] ?? '' ?></textarea>
     </div>
     <div class="create-listing-options-grid my-3 mb-3  d-flex justify-content-between">
         <div class="exchange-method-option">
             <table class="table table-borderless">
                 <tbody>
-                    <tr>
+                <tr>
+                    <?php if(!isset($params['listingType']) || $params['listingType'] !== 'sell'): ?>
```

Comment:
When dynamically setting checked, we shouldn't have the entire line be part of the if statement, as if we make changes to it later it's easier to miss one of the optional ones and have inconsistencies. Only have `checked` be in the if. Ternary operators are helpful here, e.g. 
```php
<?= isset($params['listingType']) && $params['params'] === sell ? 'checked' : '' ?>
```
This block returns 'checked' if the condition is true, and nothing otherwise.